src/test/java/edu/kis/vh/nursery/collections/IntLinkedListTest.java

```diff
@@ -26,7 +25,7 @@ public void isFull() {
 
     @Test
     public void top() {
-        assertEquals(list.top(),-1);
+        assertEquals(list.top(),0);
```

Comment:
The original constant should be used (it would be best to use a common constant in collections interface).